### ISSUE TITLE
18F needs to standardize how Project Health is done

### ISSUE THREAD

**Issue by kfoley-18F** [18F/delivery-assurance#79]
### A description of the work
Project Health has been done differently by different 18F teams, and this means that the data collected isn't consistent or useful for the intended outcomes. LT members and AMs have tried to rework PH, but this hasn't landed. 
We need a single way to collect Project Health information on a consistent reporting cadence across 18F teams (the start of "the 18F Way") so that all the stakeholders for PH information find it valuable and so that we have one process to iterate on instead of reinventing wheels. 
We also need a plan to communicate and roll this out as change management across 18F. 

### Point of contact on this issue
@SelenaJV  and Erica Perkins

### Reproduction steps (if necessary)
DA Team couldn't figure out how to do PH among the myriad ways already present in 18F. 
LT already struggles to understand and make good use of PH reported data. 
This process isn't working for anybody. 
We need to fix it. 

**Billable?**

- [ ] Yes
- [X] No
If yes, tock code:

**Skills needed**

- [X] Any human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [X] Other: Account Management

This should be a multi-person project with a mix of experience and disciplines. 

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [X] No

How much time do you anticipate this work taking?
Likely a full 2 weeks. 

### Acceptance Criteria

- [X] What does "done" look like?

As an 18F project team member, an 18F LT member, an 18F AM, and the Chief of Delivery, I know and follow the one consistent way that Project Health information is collected and reported, and PH purpose and stakeholder needs are clearly documented and understood around 18F. 
At the end of this iteration, documentation exists that describes recommendations for how 18F should change/modify/implement Project Health tracking consistently, including some recommendations for change management and comms to the rest of 18F. 
Be prepared to brief LT on the recommendations, present at Team Coffee, and possibly implement them in future iterations, so think through who on DA team needs to carry forward context.

**Comment by colinmurphy01** [comment]
This feels like it _could_ be an Epic. 

Could I suggest testing [the template out](https://docs.google.com/document/d/1daGZGh18V8ZZPY0k3YNPpL5OJ7nBdI2FdKsEOwx83OI/edit#) and then I'm actively figuring out the process for governance on this type of stuff.

**Comment by janelx** [comment]
This feels like at least 2 parts. 1. Creating recommendations and sharing with LT. 2. Carrying out and sharing the new template) 

Proposed actions for completing 1st portion:
- Interview with Selena and/or Erica on desired goals, needs and to identify key stakeholders. Validate/invalidate any assumptions. 
- Review existing project health
- Redesign and document recommendations
- Review with Selena and/or Erica

**Comment by annepetersen** [comment]
From today's meeting discussion, some notes:

Needs for project health: what do teams need to request help; what do AM need to check their project and flag when things need help; what does LT need to see what’s on fire and how they can be useful. Make sure to check with all reps on this. Could put a request out for input. The value of it organizationally for 18F.

More experienced AMs may have stronger opinions: more voices may be better

**Comment by annepetersen** [comment]
Requested access 👀

**Comment by janelx** [comment]
Main point of contact is traveling the next week and has requested that we put this down and look at picking it back up in 2 weeks (4/10/2023). 

So far completed:
[Planning doc](https://docs.google.com/document/d/1hZAmc3sEBsgwErqq37qyvmLvcrPA25SDnQ-e430FNSo/edit#heading=h.eetmedgoxhud):
- Review and synthesis of current project health and feedback - resources and people I spoke to at the end of the doc
- Proposed deliverables 
Shared planning doc to review with Selena and Erica - they've done a preliminary scan of the doc but have requested more time (not approved yet)
[Sketch of updated Project Health form](https://docs.google.com/document/d/1FxJFvS4O36awr0j90DabBvohDJjWkgincxFF_1EykZM/edit#)


To do:
- [ ] Clean up docs to be able to handoff to a new DA team members if need
- [ ] Create bite-sized tickets for project deliverables

**Comment by MelissaBraxton** [comment]
@SelenaJV  - Do you want us to pick up work on this again?

**Comment by alexbielen** [comment]
Notes from refinement: overlaps with the Epic Proposal. Erica and Selena are both the co-owners of this work. Alex will follow up with Selena and Erica

**Comment by SelenaJV** [comment]
Yes. I think we can restart this card this week or next. Let me coordinate with @Erica-Perkins and get back to you with which one of us should be the owner.

**Comment by MelissaBraxton** [comment]
@annepetersen and @janelx - I added a task list to clarify the scope of this issue.

**Comment by janelx** [comment]
Here are the proposed steps to completing this issue. More [details here.](https://docs.google.com/document/d/1hZAmc3sEBsgwErqq37qyvmLvcrPA25SDnQ-e430FNSo/edit#heading=h.k26ppd3oa7vt)
**Continued discovery:**
- Learn more about project leadership presentation (Colin Murphy)
- Learn more about project phases (Laura Gerhardt)

**Document/Design:** 
- Update Project Health Form- based on feedback ([currently working on/ a work in progress])
- Update Project Health Readme / combine w/form
- Document different ways to facilitate answer project health
- Update airtable views/create a dashboard - consult w/Kelsey (SME)
- Review form: Share  and gather feedback from Selena, Erica P., & account managers

**Feedback/Test:**
- Stakeholder review
- Present Project Leadership Guild for feedback
- Pilot with a few a teams

**Communicate:**
- Present recommendations to leadership on how to communicate project health updates to standardize and make “official”

**Comment by MelissaBraxton** [comment]
Once we have a new issue to cover next steps, we feel free to close this out, @janelx!
